bt_lession1/Ex4/ex4.js

```diff
@@ -1,6 +1,6 @@
 /** @format */
 
-// Tìm số lớn nhất trong hai số
+// Tìm số lớn nhất trong hai số nè
 // + Data 1: a = 12, b = -9
 // + Data 2: a = 6, b = 6
 // + Data 3: a = -189, b = -987
```

Comment:
sao lại làm phức tạp thế này?
- Thuật toán có vẻ đang sai nhé, mặc dù kết quả đúng 
- Tại sao không dùng let largest
- Làm lại